staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -129,7 +129,22 @@ func (a *AdmissionOptions) ApplyTo(
 }
 
 func (a *AdmissionOptions) Validate() []error {
+	if a == nil {
+		return nil
```

Comment:
Should not return nil, which will raise exception. `len(nil)` is invalid.

Move `errs := []error{}` up and return `errs` instead.

Reply:
make sense

Reply:
I just write a demo . Test result show this style is ok.

Reply:
Actually should remove `if a== nil`. It won't be nil. Right?

Reply:
return value is not just nil.  It is an empty slice.

Reply:
```
// Validate checks invalid config combination
func (o *AuditOptions) Validate() []error {
	if o == nil {
		return nil
	}

	allErrors := []error{}
```
This is a safe way.

Reply:
Prevent if just declare a var without initialize may panic.
```
var opt *AdmissionOptions
opt.Validate()
```

Reply:
I mean you should remove

```
if a== nil {
xxxx
}
```

Reply:
> return value is not just nil. It is an empty slice.

It's a typed nil of slice type ;-)

Reply:
> I mean you should remove

I think it's fine and correct to keep.

Reply:
Professional name typed nil.